public/js/controllers/ui_controller.js

```diff
@@ -189,28 +189,47 @@ export class UIController {
         UIController.resetUI();
         document.head.appendChild(UIController.getScrollJsScript());
         let html = `<div class="container"> 
-            <div class="d-md-flex justify-content-between p-1">
+            <div class="d-md-flex justify-content-between">
+                <div class="d-flex">
+                    <span class="font-weight-bold txt-brown mr-1">Africa's Talking Units Balance:</span>
+                    <div class="font-weight-bold mb-0" id="AT-units-bal"></div>
```

Comment:
Moving this to the top is a good idea, however, the UI now looks a bit untidy and congested. I would create a new card above with three columns and populate them with;
- Project Name 
- Africas Talking Bal
- Last Updated time
The balance the spacing between the time filters and the above card.

Reply:
I've done some changes, suggestions/feedback are welcome.